docs/tutorials/learn.md

```diff
@@ -569,7 +569,7 @@ This is not acceptable if you care about performance. In an imperative
 language, one would use a simple loop going over the list without any
 memory allocation. In pseudocode:
 
-```juvix
+```
```

Comment:
Maybe adding `pascal` or something like this here would give reasonable highlighting? Do we have access to predefined highlighting for other languages?